### ISSUE TITLE
Add support for ES module syntax (ESM)

### ISSUE THREAD

**Issue by jorgecasar** [catdad/canvas-confetti#56]
[ES module syntax (ESM)](http://2ality.com/2014/09/es6-modules-final.html) is JavaScript’s modern native module system.

Tools like [Pika](https://www.pikapkg.com/about/) can help.

**Comment by catdad** [comment]
Thanks for submitting this issue! However, instead of linking to a packager, can you detail the use case that does not work for you, or requirements that are not being met? That would be really useful

**Comment by jorgecasar** [comment]
Modern browsers support ES module syntax and I would like to use llike:

````
<script type="module">
import { confetti } from 'canvas-confetti';
confetti();
</script>
```

**Comment by catdad** [comment]
@jorgecasar I put together a beta of an es module that has been published now. From the CDN, I use it like this:

```html
<script type="module">
  import confetti, { create } from 'https://cdn.jsdelivr.net/npm/canvas-confetti@0.2.0-beta0/dist/confetti.module.mjs';

  // global confetti function
  confetti();

  // custom canvas
  const canvas = document.createElement('canvas');
  canvas.style.width = '100%';
  canvas.style.height = '100%';

  document.body.appendChild(canvas);

  const myConfetti = create(canvas, { resize: true });
  myConfetti();
</script>
```

This has been working for me, but let me know what you think. If all is well, I will publish the official version of this as well.

**Comment by jorgecasar** [comment]
I will try it tommorow.

Remember to change the package.json to point the module version to this file.

In the other hand, I'm not very fan of the use of mjs extension,  because it's JavaScript and it's the new way to write JavaScript. I prefer to rewrite the old one or just put the "module" sufix (`confetti.module.js`).

Thanks for the quick response and solve the issue.

**Comment by jorgecasar** [comment]
I've just try and it works perfectly. Thanks a lot!

**Comment by catdad** [comment]
Published in 0.2.0
